### PR TITLE
Design updates in Editing activity.

### PR DESCRIPTION
**Pull request by dbrant**
https://phabricator.wikimedia.org/T313223

### CODE REVIEW

**Comment by dbrant** [comment]
@cooltey @sharvaniharan  This is now ready. Mea culpa for the size.

**Review by cooltey — CHANGES_REQUESTED** [review]
Excellent upgrade! 💯

File: app/src/main/java/org/wikipedia/edit/EditSectionActivity.kt [hunk 1]
```diff
@@ -121,11 +133,52 @@ class EditSectionActivity : BaseActivity() {
...
+            bottomSheetPresenter.show(supportFragmentManager,
+                    LinkPreviewDialog.newInstance(HistoryEntry(PageTitle(title, pageTitle.wikiSite), HistoryEntry.SOURCE_INTERNAL_LINK), null))
+        }
+
+        override fun onRequestInsertMedia() {
+            requestInsertMedia.launch(InsertMediaActivity.newIntent(this@EditSectionActivity, pageTitle.displayText))
+        }
+
+        override fun onRequestInsertLink() {
+            val searchIntent = SearchActivity.newIntent(this@EditSectionActivity, Constants.InvokeSource.EDIT_ACTIVITY, null, true)
```
[the remark this answers is not in the document]
Comment (dbrant) [reply app/src/main/java/org/wikipedia/edit/EditSectionActivity.kt]: yep

File: app/src/main/java/org/wikipedia/edit/EditSectionActivity.kt [hunk 2]
```diff
@@ -121,11 +133,52 @@ class EditSectionActivity : BaseActivity() {
...
+            hideAllSyntaxModals()
+        }
+    }
+
     public override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         binding = ActivityEditSectionBinding.inflate(layoutInflater)
         setContentView(binding.root)
         setNavigationBarColor(ResourceUtil.getThemedColor(this, android.R.attr.colorBackground))
+        supportActionBar?.elevation = DimenUtil.dpToPx(4f)
```
[the remark this answers is not in the document]
Comment (dbrant) [reply app/src/main/java/org/wikipedia/edit/EditSectionActivity.kt]: yes!

File: app/src/main/java/org/wikipedia/theme/ThemeChooserDialog.kt [hunk 3]
```diff
@@ -286,24 +304,36 @@ class ThemeChooserDialog : ExtendedBottomSheetDialogFragment() {
...
-                FontSizeAction.DECREASE -> {
-                    app.setFontSizeMultiplier(Prefs.textSizeMultiplier - 1)
+            val currentMultiplier: Int
+            if (isEditing) {
+                currentMultiplier = Prefs.editingTextSizeMultiplier
+                Prefs.editingTextSizeMultiplier = app.constrainFontSizeMultiplier(when (action) {
+                    FontSizeAction.INCREASE -> currentMultiplier + 1
+                    FontSizeAction.DECREASE -> currentMultiplier - 1
+                    FontSizeAction.RESET -> 0
+                })
```
[the remark this answers is not in the document]
Comment (dbrant) [reply app/src/main/java/org/wikipedia/theme/ThemeChooserDialog.kt]: yes

**Pull request merged**